src/plays/motivate-me/components/cards/index.css

```diff
@@ -0,0 +1,41 @@
+.card-container-main{
```

Comment:
@Hat52, Add prefix to your CSS classes so that it won't conflict with other play CSS classes.

Please follow the same structure everywhere.

```suggestion
.motivation_quote_card-container-main{
```

Reply:
@Hat52 , Did you change this as well?

Reply:
yes, I have changed this as well.